qa/pom.xml

```diff
@@ -103,17 +103,17 @@
       <groupId>org.jboss.openjdk-orb</groupId>
       <artifactId>openjdk-orb</artifactId>
       <version>${version.org.jboss.openjdk-orb}</version>
-      <classifier>sources</classifier>
     </dependency>
     <dependency>
       <groupId>org.jboss.openjdk-orb</groupId>
       <artifactId>openjdk-orb</artifactId>
       <version>${version.org.jboss.openjdk-orb}</version>
+      <classifier>sources</classifier>
     </dependency>
     <dependency>
-      <groupId>org.jacorb</groupId>
-      <artifactId>jacorb-omgapi</artifactId>
-      <version>${version.org.jacorb.jacorb}</version>
+      <groupId>org.jboss.openjdk-orb</groupId>
```

Comment:
Doesn't this duplicate the dependency around line 103?

Reply:
I am not following what is happening here. The `git diff` reported here come from [this](https://github.com/jbosstm/narayana/blob/c0a476fa03c762e4f5b98cf0eab332617985c61d/qa/pom.xml) commit whereas the commit in [my branch](https://github.com/jmfinelli/narayana/blob/JBTM-3577/qa/pom.xml) shows that there are no duplications. Do you have any idea why, @tomjenkinson?

Reply:
Strangely enough, it got `outdated` after my last rebase